main.js

```diff
@@ -405,75 +450,74 @@ function round(number) {
 }
 
 function activity(string) {
-	dc?.user ?.setActivity(string);
+	dc?.user?.setActivity(string);
 }
 
 //the discordBot part starts here.
 
 function userInput(cmd, DiscordOrigin, discordMsg, channel) {
+	let splitCmd = cmd.split(" ")
 	cmd = cmd.toLowerCase();
+	if (cmd.includes(" ")) {
+		cmd = cmd.split(" ")[0]
+	}
 
 	switch (cmd) {
 		case "help":
 		case "commands":
-			console.log(" help: Lists available commands.");
-			console.log(" start 14:00: Start queue at 2pm.");
-			console.log(" play 8:00: Tries to calculate the right time to join so you can play at 8:00am.");
-			console.log(" start: Starts the queue.");
-			console.log(" loop: Restarts the queue if you are not connect at the end of it");
-			console.log(" loop status: Lets you know if you have reconnect on or off.")
-			console.log(" update: Sends an update to the current channel with your position and ETA.");
-			console.log(" url: displays the github url");
-			console.log(" stop: Stops the queue.");
-			console.log(" exit or quit: Exits the application.");
-			console.log(" stats: Displays your health and hunger.");
+			msg(DiscordOrigin, discordMsg, "Help", " help: Lists available commands.\n start 14:00: Start queue at 2pm.\n play 8:00: Tries to calculate the right time to join so you can play at 8:00am.\n start: Starts the queue.\n loop: Restarts the queue if you are not connect at the end of it,\n loop status: Lets you know if you have reconnect on or off.\n update: Sends an update to the current channel with your position and ETA.\n url: displays the github url.\n stop: Stops the queue.\n exit or quit: Exits the application.\n stats: Displays your health and hunger.");
 			break;
+		case "status":
 		case "stats":
 			try {
-			if (conn.bot.health == undefined && conn.bot.food == undefined){
-			console.log("Unknown.")
-			break;}
-			else
-			{if (conn.bot.health == 0)
-			console.log("Health: DEAD");
-			else
-			console.log("Health: " + Math.ceil(conn.bot.health)/2 + "/10");
-			if (conn.bot.food == 0)
-			console.log("Hunger: STARVING");
-			else
-			console.log("Hunger: " + conn.bot.food/2 + "/10");}
-			} catch (err)
-			{console.log(`Start 2B2W first with "Start".`)}
+				if (conn.bot.health == undefined && conn.bot.food == undefined) {
```

Comment:
Discussion:
i'm fine with chat commands going away provided they are available in slash command form. less to  maintain.  otherwise, consider calling the actual  command code here, instead of rewriting it?